pkg/agent/syssetup/setup.go

```diff
@@ -3,11 +3,19 @@
 package syssetup
 
 import (
-	"io/ioutil"
 	"os"
 	"os/exec"
+	"runtime"
 
+	"github.com/google/cadvisor/machine"
+	"github.com/google/cadvisor/utils/sysfs"
 	"github.com/sirupsen/logrus"
+	"k8s.io/kubernetes/pkg/util/sysctl"
+)
+
+const (
+	ConntrackMaxPerCore = 32768
```

Comment:
Do these need to be exported?

Reply:
I predicting that I will need them for RKE2's kube-proxy chart but I am not sure until I get over there.